src/RoadRegistry.BackOffice.UI/src/DownloadExtract.elm

```diff
@@ -36,7 +36,7 @@ type alias Model =
 
 init : String -> ( Model, Cmd Msg )
 init url =
-    ( { header = Header.init |> Header.downloadBecameActive
+    ( { header = Header.init |> Header.downloadExtractBecameActive
```

Comment:
I'd prefer if `extract` was replaced with `editor` for now, as a term. Makes it consistent across the board even if not the best of names.

Reply:
`downloadForEditor` in that case?

Reply:
or just `downloadEditor`?

Reply:
`downloadForEditor` has a nice ring to it.